test/token/ERC20/BurnableToken.behaviour.js

```diff
@@ -0,0 +1,40 @@
+import assertRevert from '../../helpers/assertRevert';
+
+export default function ([owner]) {
+  describe('basic burnable token', function () {
+    const from = owner;
+
+    describe('when the given amount is not greater than balance of the sender', function () {
+      const amount = 100;
+
+      it('burns the requested amount', async function () {
+        await this.token.burn(amount, { from });
+
+        const balance = await this.token.balanceOf(from);
+        assert.equal(balance, 900);
+      });
+
+      it('emits a burn event', async function () {
+        const { logs } = await this.token.burn(amount, { from });
+        const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+        assert.equal(logs.length, 2);
+        assert.equal(logs[0].event, 'Burn');
```

Comment:
If we're abstracting this into a behavior, I think we need to stop relying on the exact index of the log. We have a helper [`expectEvent.inLogs`](https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/test/helpers/expectEvent.js) that may be useful if it's modified to return the found event.